skfda/representation/basis.py

```diff
@@ -324,6 +324,42 @@ def _to_R(self):
     def inner_product(self, other):
         return numpy.transpose(other.inner_product(self.to_basis()))
 
+    def add_same_basis(self, coefs1, coefs2):
```

Comment:
I think those methods should be private. There is no reason for users to mess with them.